docs-new/docs/graphql/core/getting-started/index.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Getting started
+description: Get started with Hasura
+keywords:
+  - hasura
+  - docs
+  - start
+slug: index
+---
+
+# Getting Started
+
+## Introduction
+
+To use the Hasura GraphQL engine, you need to:
+
+- Deploy the Hasura GraphQL engine, with access to a Postgres database
+  to store its metadata.
+- Connect new/existing database(s) and set up and test your GraphQL
+  API using the Hasura console UI _(also possible via CLI or API)_.
+- Consume the generated GraphQL API from your client apps.
+
+## Quickstart guides
+
+- [Using Hasura Cloud](/graphql/cloud/getting-started/index.mdx#cloud-getting-started) - Create a new Hasura
+  Cloud project with just a few clicks.
+  - Recommended as it's **the fastest way** to try Hasura GraphQL
+    engine out.
+  - Comes with **extra features for reliability and security** and a
+    managed metadata database.
```

Comment:
pls fix line breaks

Reply:
I think this is supposed to be nested, I checked the previous docs, can you please confirm?

Reply:
This is nested. I was referring to the content text itself. Only 1 or two words in second line even if first line is super short.

```suggestion
  - Recommended as it's **the fastest way** to try Hasura GraphQL engine out.
  - Comes with **extra features for reliability and security** and a managed metadata database.
```